src/package/package.ts

```diff
@@ -6,7 +6,7 @@
  */
 import { Messages, sfdc } from '@salesforce/core';
 import { AsyncCreatable, Duration } from '@salesforce/kit';
-import { QueryResult } from 'jsforce';
+import { QueryResult, SaveResult } from 'jsforce';
```

Comment:
Point for discussion. I aliased jsforce SaveResult with `PackageSaveResult` to avoid having to import jsforce on the client side. Good idea or bad? @WillieRuemmele @shetzel